grammar update to vault docs

bhanuvrat opened the pull request:
grammar nazi rearrangements.

bhanuvrat committed:
grammatical rearrangements.

bhanuvrat committed:
[grammar nazi] rearrangment.

erikdw commented on docsite/rst/playbooks_vault.rst:
```diff
@@ -14,7 +14,7 @@ What Can Be Encrypted With Vault
 The vault feature can encrypt any structured data file used by Ansible.  This can include "group_vars/" or "host_vars/" inventory variables, variables loaded by "include_vars" or "vars_files", or variable files passed on the ansible-playbook command line with "-e @file.yml" or "-e @file.json".  Role variables and defaults are also included!
 
-Because Ansible tasks, handlers, and so on are also data, these can also be encrypted with vault.  If you'd like to not betray what variables you are even using, you can go as far to keep an individual task file entirely encrypted.  However, that might be a little much and could annoy your coworkers :)
+Because Ansible tasks, handlers, and so on are also data, these too can be encrypted with vault.  If you'd not like to betray even the variables you are using, you can go as far as keeping individual task files entirely encrypted.  However, that might be a little too much and could annoy your coworkers :)
```
A couple suggestions on your suggestions:
- I find "If you'd not like to betray even the variables you are using" to be confusing compared to the original wording.  Maybe just removing "even" within the "you are even using" fragment would suffice.
- "a little much" is a common phrase, I don't see a need to change that.

Also, "patch 1" or "patch 2" as the title of the PR isn't very great, I'd change it to reflect that you're making grammar changes (as your commit messages state).

bcoca commented:
what about " If you'd like to avoid exposing even the variables ..."

abadger commented:
Maybe this would be better.
- Changes the first sentence to not lead with a conjunction
- Get rid of the "you would like to not betray" phrase as those words aren't commonly put together in English (it makes for a good phrase in fiction but something clearer is better for documentation.)

"Ansible tasks, handlers, and so on are also data so these can be encrypted with vault as well.  If you don't want to even reveal the variables you are using you can go as far as keeping individual task files entirely encrypted.  However, that might be a little too much and could annoy your coworkers :)"

erikdw commented:
Since we are all focused on the middle sentence I'll propose a complete rewrite:

"If you don't want to even reveal the variables you are using you can go as far as keeping individual task files entirely encrypted." ->

"You can even choose to hide the variables you are using by encrypting the task files in their entirety."

msabramo commented:
"To hide the names of variables that you're using, you can encrypt the task files in their entirety."

But this is just a slight tweak of @erikdw's 2nd proposal, which I also like.

erikdw commented:
Heh, been awhile since the VMware shuttle to South SJ, huh Marc?

I like your variant too, though it should end with "entirety" instead of "identity".

msabramo commented:
Ha ha. I knew your name sounded familiar. Hope you're doing well. 

Identity - awesome example of iPhone spell "correction" :-)

bhanuvrat commented:
@abadger's suggestion is appealing.
@msabramo's feels concise and to the point.
shall I make the changes?

bhanuvrat committed:
full rewrite of the paragraph.

following @abadger's suggestion[1] in the comments

[1] https://github.com/ansible/ansible/pull/11410#issuecomment-116049590

erikdw commented:
@bhanuvrat : I see the change you made to avoid starting with "Because".  Can you please also make @msabramo's suggested change?

bhanuvrat committed:
makes it more concise.

@msabramos's suggestions[1] incorporated.

[1] https://github.com/ansible/ansible/pull/11410#issuecomment-116319780

bhanuvrat commented:
@erikdw done.

abadger commented:
@bhanuvrat  @erikdw @msabramo @bcoca  : Thanks for all the work on this paragraph guys!

# Merged

Hi!

This has been merged in, and will also be included in the next major release.

If you or anyone else has any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
